src/control-plane/backend/lambda/api/common/constants.ts

```diff
@@ -62,9 +62,30 @@ const PIPELINE_SUPPORTED_REGIONS = [
   'cn-northwest-1',
 ];
 
+const SERVICE_CATALOG_SUPPORTED_REGIONS = [
```

Comment:
The backend is sharing the infra's constants via file `constants-ln`, could you move those to the infra's constants to share this definition?

Reply:
done